libs/blocks/modal/modal.css

```diff
@@ -359,6 +386,13 @@
     width: 80%;
   }
 
+  .dialog-modal.three-in-one {
```

Comment:
I see `.dialog-modal.upgrade-flow-modal` has the exact properties, can we combine them somehow?